config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -26,7 +31,7 @@ periodics:
           image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20220512-b6ea825-4.2.1
           resources:
             requests:
-              cpu: 2
+              cpu: "2"
```

Comment:
note: Making this a string is recurring too. The field can be specified as a string (e.g. `2000m`) so having it be a string everywhere makes life easier in the generation code.